scss/media.scss

```diff
@@ -0,0 +1,442 @@
+      position: relative;
+      .head-accordion {
+        justify-content: space-between;
+        .number {
+          font-size: 110px;
+          line-height: 130px;
+          width: 50px;
+          margin-right: 113px;
+        }
+        .accord-text {
+          font-size: 24px;
+          line-height: 36px;
+        }
+        .accordion-button {
+          width: 37.5px;
+          height: 35px;
+          &::after {
+            width: 20px;
+            height: 2px;
+            top: 15px;
+            left: 6px;
+          }
+          &::before {
+            width: 20px;
+            height: 2px;
+            top: 15px;
+            left: 6px;
+            transform: rotate(90deg);
+          }
+        }
+      }
+      .panel {
+        display: flex;
+        .main-icon {
+          margin: 42px 0 0 0;
+        }
+        .footer-accordion {
+          font-size: 16px;
+          margin-bottom: 63px;
+        }
+        .footer-accordion-container {
+          max-width: 500px;
+          margin-left: 135px;
+        }
+      }
+    }
+    .glossary-contaner {
+      margin-top: 160px;
+      padding-top: 19px;
+      margin-bottom: 50px;
+      #head-glossary {
+        width: 342px;
+        height: 41px;
+        margin-left: 0;
+        margin: 19px 0 50px 0;
+      }
+    }
+  }
+
+  .accordion-active {
+    width: 35px;
+    height: 35px;
+    &::after {
+      width: 20px;
+      height: 2px;
+      top: 14px;
+      left: 5px;
+    }
+  }
+
+  .accord-text-active {
+    max-width: 175px;
+    margin-top: 57px;
+    font-size: 42px;
+    line-height: 54px;
+  }
+
+  .section {
+    .section-wrapper {
+      display: flex;
+      max-width: 1440px;
+      margin: auto;
+      .slider-container {
+        margin: 60px 0 60px -10px;
+        .slider-head {
+          margin: 60px 73px 29px 151px;
+        }
+        .slider-button-container {
+          margin-right: 19px;
+          .slider-button {
+            width: 25px;
+            height: 25px;
+            margin-top: 20px;
+            &::after {
+              width: 10px;
+              height: 10px;
+              top: 7px;
+              left: 9px;
+            }
+            &::before {
+              top: 11px;
+              width: 15px;
+              height: 2px;
+            }
+          }
+        }
+      }
+      .slider-text-container {
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+
+        margin: auto;
+        p {
+          font-size: 90px;
+          line-height: 120px;
+          padding-bottom: 70px;
+          margin: 0;
+        }
+        span {
+          font-size: 24px;
+          line-height: 36px;
+        }
+        .slider-text {
+          padding-left: 40px;
+          margin-left: 80px;
+        }
+      }
+    }
+  }
+  .active-slider-link {
+    &::before {
+      width: 105px;
+      margin-right: 15px;
+    }
+  }
+
+  .faqs-container {
+    max-width: 1440px;
```

Comment:
use common class container instead, you don't need dublicate styles

Reply:
was allocated to a separate cssla